infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftProjectFactory.java

```diff
@@ -93,4 +125,58 @@ OpenShiftWorkspaceServiceAccount doCreateServiceAccount(String workspaceId, Stri
     return new OpenShiftWorkspaceServiceAccount(
         workspaceId, projectName, getServiceAccountName(), getClusterRoleName(), clientFactory);
   }
+
+  @Override
+  protected Optional<KubernetesNamespaceMeta> fetchNamespace(String name)
+      throws KubernetesClientException, InfrastructureException {
```

Comment:
Looks like `KubernetesClientException` can't actually be thrown here.

Reply:
:+1: 
fixed